### ISSUE TITLE
Readme build instructions tweaks

### ISSUE THREAD

**Issue by CodingKoopa** [docker/docker-credential-helpers#116]
I'm trying to use the helper program to use `pass` with Docker. I'm using the `armhf` architecture on Ubuntu 16.04. Only `amd64` releases are provided here, and [the Ubuntu backports for these programs](https://packages.ubuntu.com/search?suite=all&section=all&arch=any&keywords=golang-docker-credential-helpers&searchon=names) don't go far enough to Trusty, so I have to build `docker-credential-pass` myself. I'm using these commands:
- `sudo apt-get install golang-go` to install Go.
- `cd ~/Documents/ && mkdir Go` to make a working directory for Go.
- `export GOPATH=$HOME/Documents/Go` to set the working directory for Go.
- `go get github.com/docker/docker-credential-helpers` as instructed by the readme. This outputted this line on Go 1.6.2:
```
package github.com/docker/docker-credential-helpers: no buildable Go source files in /home/kyle/Documents/Go/src/github.com/docker/docker-credential-helpers
```
And on Go 1.10.3:
```
package github.com/docker/docker-credential-helpers: no Go files in /home/kyle/Documents/Go/src/github.com/docker/docker-credential-helpers
```
- `cd $GOPATH/src/github.com/docker/docker-credential-helpers` to enter the readme directory. The readme says to use `cd $GOPATH/docker/docker-credentials-helpers`, which seems to be the wrong path here.
- `make pass` to compile `docker-credential-pass`.
- `mkdir $GOPATH/bin`, `ln -s "$GOPATH/src/github.com/docker/docker-credential-helpers/bin/docker-credential-pass" $GOPATH/bin/`, and `echo "PATH=$PATH:$GOPATH/bin" >> ~/.bashrc` to add the binary to my path.

tl;dr: The command in step 1 outputted an error for me, and a command in step 2 has a different path than what I got.

**Comment by rouaks** [comment]
Same issue

**Comment by thaJeztah** [comment]
https://github.com/docker/docker-credential-helpers/pull/168 switched this project to use go modules, which should probably help with this issue.

Closing this ticket for now, but feel free to comment if there's things left to address
